pkg/operations/start.go

```diff
@@ -107,6 +107,9 @@ func StartVM(vm *api.VM, debug bool) error {
 		return fmt.Errorf("failed to start container for VM %q: %v", vm.GetUID(), err)
 	}
 
+	// Set the container ID for the VM
+	vm.Status.Runtime = &api.Runtime{ID: containerID}
```

Comment:
We'll decide on some conventions here, but probably we'll follow Kubernetes' style `docker://abcdef...`